lib/web_console/testing/erb_helper.rb

```diff
@@ -0,0 +1,24 @@
+require 'web_console/testing/fake_middleware'
+
+# This class is to pre-compile 'templates/*.erb'.
+class ERBHelper
+  def initialize(path)
+    @erb  = ERB.new(File.read path)
+    @view = FakeMiddleware.new(
+      view_path: root('lib/web_console/templates'),
+    ).view
+  end
+
+  def build
+    @erb.result binding
+  end
+
+  def root(path = '')
+    Pathname(File.expand_path '../../../../', __FILE__).join path
```

Comment:
The `gem_root` may be useful for the other extensions as well. May be worth to extract as `WebConsole.gem_root` or  similar class method.